transifex/native/django/utils/templates.py

```diff
@@ -31,7 +37,7 @@ def find_filter_identity(filter_name):
     :return: a callable that represents a filter that does not filtering
     :rtype: callable
     """
-    return lambda obj: obj
+    return identity
```

Comment:
You could also write this as `lambda *args: args[0]` instead of creating a method with a (large) fixed number of arguments.

Reply:
Great, I thought this is not working on lambda expressions. Are you sure that it works both on python2 as well on python3? @dbendilas

Reply:
@dbendilas It works!

Reply:
@dbendilas I think that for some reason does not work with django, so I am going to leave it as it is for now

Reply:
We had tried adding `*args` to the `identity` function. Django inspects the filter functions and counts the arguments provided, not `*args`.